aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
```

Comment:
Would be great to go with Itay's suggestion of using a flat JSON file without the rego to make parsing etc. easier if we're just storing metadata.